src/content/docs/explanation/what-is-daytona.mdx

```diff
@@ -1,15 +1,15 @@
 ---
 title: What is Daytona?
-description: Understand how Daytona can help you manage your Dev Environments.
+description: Understand how Daytona can help you manage your Workspaces.
 sidebar:
   label: What is Daytona?
 ---
 
 Daytona is an open-source Development Environment Manager (DEM) licensed under the Apache License 2.0.
-Daytona allows you to manage and deploy reproducible Development Environments (DEs) based on standard OCI containers, with built-in support for the Dev Container standard.
-Daytona's architecture provides the future possibility to base Dev Environments using other standards, such as Dockerfiles, Docker Compose configurations, Nix, and Devfile.
+Daytona allows you to manage and deploy reproducible Workspaces (Dev Environments) based on standard OCI containers, with built-in support for the Dev Container standard.
```

Comment:
`reproducible development environments, called Workspaces,...` might be more appropriate. Just a suggestion

Reply:
Good shout. I've improved the phrasing a bit in [c18113b](https://github.com/daytonaio/docs/pull/21/commits/c18113bfb4c984936eb183ddc0f5037d7b025652).